integration/spark/src/test/scala/org/apache/carbondata/spark/testsuite/alterTable/TestAlterTableAddColumns.scala

```diff
@@ -136,14 +139,228 @@ class TestAlterTableAddColumns extends QueryTest with BeforeAndAfterAll {
     sql(s"""DROP TABLE IF EXISTS ${ tableName }""")
   }
 
+  test("Test adding of array of all primitive datatypes") {
```

Comment:
@akkio-97 please add test case for add columns with comment for complex column and then also add a test case to verify the SchemaEvolutionEntry after add column

Reply:
done